### ISSUE TITLE
Species search for GO CAM on public website

### ISSUE THREAD

**Issue by hattrill** [geneontology/go-site#2078]
When searching species/organism on https://geneontology.cloud/home

Drosophila gets one hit
-cGAS-Sting pathway (Drosophila) 
and 
D.mel gets 9
-all with D.mel in title
and
taxid:7227
0
and
drosophila melanogaster
0

(there are 15 models in production in Noctua)

So, it's not a species search, just a string search. Below the search box is listed: Title, GO-Term (BP, MF, CC), Gene Product, Species, PMID, Curator, Group, Title, Biological Process, Molecular Function, Cellular Component, Gene Product, Group; as things that can be searched.

This is very misleading for users.

Please fix this to search the fields in the list correctly, mirroring that on the Noctua site.

**Comment by kltm** [comment]
@hattrill Yes, that is a bit of a problem. I'm not sure if there is going to be much more development on the /go-cam site at this point, but we can check in to see if there is anything we could hack in to smooth some of this over.
The current roadmap has it being replaced in the not-so-far future with complete integration of GO-CAMs into AmiGO, removing the need for a separate GO-CAM site.

**Comment by hattrill** [comment]
A bit of a hack would be good or may be just update the list of things that can be searched to reflect what is being searched.
